test/controllers/users_controller_test.rb

```diff
@@ -1,4 +1,30 @@
 require "test_helper"
 
 describe UsersController do
+
```

Comment:
Just noting some tests, like logout and login with invalid auth params, missing.